recipes/libhdfs3/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "2.2.31" %}
+
+package:
+  name: libhdfs3
+  version: "{{ version }}"
+
+source:
+    git_url: https://github.com/PivotalRD/libhdfs3.git
+    git_tag: v{{ version }}
+
+build:
+  number: 0
+  # This package is a dependency of hdfs3, which is primarily used with HDFS on Linux.
```

Comment:
@ocefpaf I requested it.  I think it's good to have that, rather than having someone wonder "might OS X work, or is it a can of worms?"